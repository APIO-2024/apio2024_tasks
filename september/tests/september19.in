txy4h26c1rm1uv8tr3eonahd67u8h56x
53
10 1
0 1 2 3 4 5 6 7 8
9 8 7 6 5 4 2 3 1
10 1
0 1 2 3 4 5 6 7 8
9 7 6 8 4 3 2 1 5
10 1
0 1 2 3 4 5 6 7 8
9 8 7 6 5 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
9 8 6 7 5 3 4 2 1
10 1
0 1 2 3 4 5 6 7 8
9 7 8 5 6 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
9 8 7 6 5 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
9 7 8 6 4 5 3 1 2
10 1
0 1 2 3 4 5 6 7 8
8 7 9 6 4 3 5 1 2
10 1
0 1 2 3 4 5 6 7 8
9 7 6 8 5 3 4 1 2
10 1
0 1 2 3 4 5 6 7 8
8 9 6 5 7 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
8 9 6 7 4 5 3 1 2
10 1
0 1 2 3 4 5 6 7 8
8 9 6 7 5 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
9 7 8 6 5 3 4 1 2
10 1
0 1 2 3 4 5 6 7 8
8 9 7 6 5 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
9 7 8 6 4 5 3 2 1
10 1
0 1 2 3 4 5 6 7 8
9 8 6 7 5 4 2 3 1
10 1
0 1 2 3 4 5 6 7 8
9 7 6 5 8 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
8 9 7 6 5 4 3 1 2
10 1
0 1 2 3 4 5 6 7 8
9 8 6 7 5 4 3 1 2
10 1
0 1 2 3 4 5 6 7 8
8 9 7 6 4 3 5 1 2
40 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38
39 37 36 38 35 33 32 31 30 34 28 27 29 26 24 23 25 21 22 19 20 17 16 18 15 13 12 14 10 11 9 7 8 6 4 5 3 1 2
40 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38
38 39 37 35 36 33 34 31 30 32 29 27 28 26 24 25 23 22 20 21 18 19 17 15 16 14 13 11 12 9 8 10 6 7 4 5 2 3 1
40 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38
38 39 37 36 35 33 34 32 31 29 30 28 27 25 26 24 23 22 21 20 19 18 17 16 15 14 12 13 11 10 9 8 7 6 4 5 3 2 1
40 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38
39 38 37 36 35 34 33 32 31 30 29 28 27 26 25 24 22 23 21 20 19 18 17 16 15 14 13 12 11 10 9 8 7 6 5 4 3 2 1
40 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38
39 37 38 36 35 33 34 32 31 30 29 28 27 25 26 23 24 21 20 22 19 17 18 16 15 14 12 13 11 10 8 7 9 5 6 3 4 2 1
100 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 71 72 73 74 75 76 77 78 79 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 96 97 98
99 98 96 97 95 94 92 93 91 90 88 89 87 86 84 85 82 83 81 79 80 78 77 76 74 75 73 71 72 70 69 68 67 66 64 65 62 63 61 60 59 58 57 56 55 54 53 52 51 50 49 48 47 45 46 43 44 42 41 40 39 37 36 38 35 34 32 33 31 30 28 29 26 27 25 24 22 23 21 20 19 18 16 17 15 14 12 11 13 10 9 7 6 8 5 4 3 1 2
1000 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 71 72 73 74 75 76 77 78 79 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 96 97 98 99 100 101 102 103 104 105 106 107 108 109 110 111 112 113 114 115 116 117 118 119 120 121 122 123 124 125 126 127 128 129 130 131 132 133 134 135 136 137 138 139 140 141 142 143 144 145 146 147 148 149 150 151 152 153 154 155 156 157 158 159 160 161 162 163 164 165 166 167 168 169 170 171 172 173 174 175 176 177 178 179 180 181 182 183 184 185 186 187 188 189 190 191 192 193 194 195 196 197 198 199 200 201 202 203 204 205 206 207 208 209 210 211 212 213 214 215 216 217 218 219 220 221 222 223 224 225 226 227 228 229 230 231 232 233 234 235 236 237 238 239 240 241 242 243 244 245 246 247 248 249 250 251 252 253 254 255 256 257 258 259 260 261 262 263 264 265 266 267 268 269 270 271 272 273 274 275 276 277 278 279 280 281 282 283 284 285 286 287 288 289 290 291 292 293 294 295 296 297 298 299 300 301 302 303 304 305 306 307 308 309 310 311 312 313 314 315 316 317 318 319 320 321 322 323 324 325 326 327 328 329 330 331 332 333 334 335 336 337 338 339 340 341 342 343 344 345 346 347 348 349 350 351 352 353 354 355 356 357 358 359 360 361 362 363 364 365 366 367 368 369 370 371 372 373 374 375 376 377 378 379 380 381 382 383 384 385 386 387 388 389 390 391 392 393 394 395 396 397 398 399 400 401 402 403 404 405 406 407 408 409 410 411 412 413 414 415 416 417 418 419 420 421 422 423 424 425 426 427 428 429 430 431 432 433 434 435 436 437 438 439 440 441 442 443 444 445 446 447 448 449 450 451 452 453 454 455 456 457 458 459 460 461 462 463 464 465 466 467 468 469 470 471 472 473 474 475 476 477 478 479 480 481 482 483 484 485 486 487 488 489 490 491 492 493 494 495 496 497 498 499 500 501 502 503 504 505 506 507 508 509 510 511 512 513 514 515 516 517 518 519 520 521 522 523 524 525 526 527 528 529 530 531 532 533 534 535 536 537 538 539 540 541 542 543 544 545 546 547 548 549 550 551 552 553 554 555 556 557 558 559 560 561 562 563 564 565 566 567 568 569 570 571 572 573 574 575 576 577 578 579 580 581 582 583 584 585 586 587 588 589 590 591 592 593 594 595 596 597 598 599 600 601 602 603 604 605 606 607 608 609 610 611 612 613 614 615 616 617 618 619 620 621 622 623 624 625 626 627 628 629 630 631 632 633 634 635 636 637 638 639 640 641 642 643 644 645 646 647 648 649 650 651 652 653 654 655 656 657 658 659 660 661 662 663 664 665 666 667 668 669 670 671 672 673 674 675 676 677 678 679 680 681 682 683 684 685 686 687 688 689 690 691 692 693 694 695 696 697 698 699 700 701 702 703 704 705 706 707 708 709 710 711 712 713 714 715 716 717 718 719 720 721 722 723 724 725 726 727 728 729 730 731 732 733 734 735 736 737 738 739 740 741 742 743 744 745 746 747 748 749 750 751 752 753 754 755 756 757 758 759 760 761 762 763 764 765 766 767 768 769 770 771 772 773 774 775 776 777 778 779 780 781 782 783 784 785 786 787 788 789 790 791 792 793 794 795 796 797 798 799 800 801 802 803 804 805 806 807 808 809 810 811 812 813 814 815 816 817 818 819 820 821 822 823 824 825 826 827 828 829 830 831 832 833 834 835 836 837 838 839 840 841 842 843 844 845 846 847 848 849 850 851 852 853 854 855 856 857 858 859 860 861 862 863 864 865 866 867 868 869 870 871 872 873 874 875 876 877 878 879 880 881 882 883 884 885 886 887 888 889 890 891 892 893 894 895 896 897 898 899 900 901 902 903 904 905 906 907 908 909 910 911 912 913 914 915 916 917 918 919 920 921 922 923 924 925 926 927 928 929 930 931 932 933 934 935 936 937 938 939 940 941 942 943 944 945 946 947 948 949 950 951 952 953 954 955 956 957 958 959 960 961 962 963 964 965 966 967 968 969 970 971 972 973 974 975 976 977 978 979 980 981 982 983 984 985 986 987 988 989 990 991 992 993 994 995 996 997 998
999 997 998 996 995 994 993 991 992 990 989 988 987 986 985 983 984 981 982 980 979 977 978 976 974 975 973 971 970 972 969 968 967 966 965 964 963 962 961 960 959 958 957 956 955 954 953 951 952 950 949 948 946 947 944 943 945 942 941 940 939 938 936 937 935 934 933 931 932 929 930 928 927 926 925 924 923 921 922 920 919 918 917 916 915 914 913 911 912 910 909 908 907 906 905 903 904 902 901 899 900 898 897 896 895 894 893 892 891 890 889 888 887 886 884 885 883 882 880 881 878 879 877 875 876 874 873 872 870 871 869 868 867 866 865 864 863 862 861 859 860 858 857 855 856 854 853 852 851 850 849 848 846 847 845 844 843 842 841 840 838 839 837 836 835 834 833 832 831 830 829 828 826 827 825 824 822 823 821 819 820 818 817 816 815 814 813 812 811 810 808 809 806 807 805 803 804 802 801 800 798 799 796 797 794 795 793 792 791 790 789 787 788 786 785 784 783 782 781 779 780 778 777 776 775 774 773 771 772 770 769 768 767 766 765 764 763 762 761 760 759 758 756 757 755 754 753 751 752 750 749 748 746 747 745 744 743 741 742 740 739 738 737 736 735 733 734 732 730 731 729 728 727 725 726 723 724 722 721 720 719 718 717 716 714 715 713 712 711 710 708 709 707 706 705 703 704 702 701 700 699 698 697 696 694 693 695 691 692 689 690 688 687 685 686 684 683 682 681 679 680 678 677 676 675 674 672 673 671 670 669 668 667 666 665 664 663 662 661 660 659 658 657 656 655 654 652 653 651 649 650 648 647 645 646 644 642 643 641 640 639 638 637 636 634 635 633 632 631 630 629 628 626 627 625 624 623 621 622 620 619 618 617 616 614 615 613 612 611 610 609 608 606 607 605 604 603 602 601 600 599 598 597 596 595 594 593 592 591 590 588 589 586 587 585 584 582 583 581 579 580 578 577 576 575 573 574 572 571 570 569 568 566 567 565 564 563 562 561 560 558 559 557 556 555 553 554 552 551 550 549 548 547 546 545 544 543 542 541 540 539 538 536 537 535 534 533 532 530 531 529 528 527 526 525 523 524 522 521 520 519 518 516 517 515 514 513 512 511 510 509 508 507 506 505 504 503 502 500 501 499 498 497 496 495 494 493 492 491 490 489 488 487 486 485 484 483 481 482 480 479 478 477 476 474 475 472 473 470 471 469 468 466 467 465 464 462 463 461 460 459 458 457 456 454 453 455 452 451 450 448 449 447 446 444 445 442 443 441 440 439 438 437 436 434 435 433 432 431 430 429 428 427 425 426 424 422 423 421 419 420 418 417 416 415 414 413 412 411 410 409 408 407 406 405 404 403 402 401 400 399 398 396 397 395 394 393 392 391 390 389 388 387 386 385 383 384 382 380 381 379 377 378 375 376 374 373 371 372 369 370 367 368 366 365 363 364 362 361 359 360 358 357 356 355 353 354 352 351 350 348 349 346 347 345 344 343 342 341 340 338 339 337 336 335 334 333 332 331 330 329 328 327 325 326 324 323 322 321 320 319 317 318 315 314 316 313 311 312 310 309 308 306 307 304 305 303 302 301 300 298 299 296 297 295 293 294 292 291 290 288 289 286 287 285 284 282 283 281 279 280 277 276 278 275 274 273 271 272 270 268 269 267 265 266 264 263 261 262 260 259 258 257 256 255 253 254 252 251 250 249 248 247 246 245 244 243 242 241 240 239 238 237 236 235 234 233 232 231 230 229 228 227 226 225 224 223 222 221 219 220 218 217 216 215 214 213 212 211 209 210 208 207 205 206 203 204 202 200 201 199 198 197 195 196 194 193 192 191 190 189 188 187 186 184 185 183 181 182 180 179 177 178 176 175 174 172 173 171 170 169 168 167 166 165 163 164 162 161 160 159 157 158 156 155 154 153 152 150 151 149 148 147 146 145 144 143 142 141 139 140 137 138 136 135 134 133 132 130 131 129 128 127 126 125 123 124 122 121 119 120 118 117 116 115 114 113 112 111 110 108 109 107 106 104 103 105 102 101 100 99 98 97 96 95 94 93 92 91 90 89 88 86 87 85 84 83 82 80 81 79 78 77 75 76 74 73 72 71 70 69 68 67 66 65 64 63 62 61 59 60 58 57 55 56 53 54 51 52 50 49 48 47 46 45 44 43 42 41 39 40 38 37 36 35 34 33 32 31 30 29 27 26 28 25 24 22 23 21 20 18 19 17 15 16 14 13 12 11 10 9 7 8 5 6 4 3 2 1
100 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 71 72 73 74 75 76 77 78 79 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 96 97 98
99 97 98 96 95 94 92 93 91 90 89 88 87 86 85 84 82 83 81 80 79 78 77 75 76 74 72 73 71 70 68 69 67 65 66 64 63 62 61 60 59 58 56 55 57 53 54 52 50 51 48 49 47 46 45 43 44 42 41 40 39 38 37 36 35 34 32 33 31 30 29 28 26 27 24 25 23 22 20 21 19 18 17 16 15 14 12 13 11 10 9 7 8 6 5 4 3 2 1
40 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38
38 37 36 35 39 34 33 32 31 29 28 30 27 25 26 23 24 22 21 20 18 17 19 16 14 15 13 11 10 12 8 9 7 5 4 3 6 1 2
40 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38
38 39 37 35 36 34 32 33 31 30 28 29 26 27 25 24 22 23 21 20 19 18 17 16 15 13 14 12 11 10 9 7 8 6 5 3 4 2 1
40 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38
39 38 37 35 36 34 32 33 31 30 29 28 27 26 25 24 23 22 21 19 20 18 16 17 14 15 13 12 11 10 9 8 7 6 5 4 3 2 1
40 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38
39 37 38 35 34 36 32 33 30 29 28 27 26 25 24 31 23 21 20 19 18 17 22 15 14 16 13 12 11 10 8 7 9 6 5 3 2 4 1
40 1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38
39 37 38 36 35 34 33 31 32 30 29 28 26 27 25 24 22 23 21 20 19 18 16 17 15 13 12 14 11 10 9 8 7 6 4 3 5 2 1
10 1
0 1 2 3 4 5 6 7 8
8 9 6 7 5 4 3 1 2
10 1
0 1 2 3 4 5 6 7 8
8 7 6 5 9 3 4 1 2
10 1
0 1 2 3 4 5 6 7 8
8 9 7 6 5 4 2 3 1
10 1
0 1 2 3 4 5 6 7 8
9 8 7 5 6 4 2 3 1
10 1
0 1 2 3 4 5 6 7 8
8 9 7 6 4 5 3 1 2
10 1
0 1 2 3 4 5 6 7 8
8 9 7 6 5 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
9 8 6 5 7 3 2 4 1
10 1
0 1 2 3 4 5 6 7 8
8 7 9 6 4 5 2 1 3
10 1
0 1 2 3 4 5 6 7 8
9 8 7 6 5 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
9 7 8 6 5 4 2 3 1
10 1
0 1 2 3 4 5 6 7 8
9 8 6 7 5 4 3 1 2
10 1
0 1 2 3 4 5 6 7 8
9 7 8 5 6 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
9 8 7 5 6 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
8 7 6 9 4 5 2 3 1
10 1
0 1 2 3 4 5 6 7 8
8 9 7 6 5 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
8 9 6 7 5 3 4 2 1
10 1
0 1 2 3 4 5 6 7 8
8 7 9 5 6 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
9 8 6 7 5 4 3 2 1
10 1
0 1 2 3 4 5 6 7 8
9 8 7 6 4 5 3 2 1
10 1
0 1 2 3 4 5 6 7 8
9 8 7 6 5 3 4 2 1
