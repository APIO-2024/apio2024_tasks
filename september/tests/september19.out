7ckgnn4wyi495puj3ibqf81dqvapyv6b
8
3
9
7
7
9
6
4
5
6
5
7
6
8
7
7
6
7
7
5
19
23
33
38
27
75
833
80
21
30
34
17
30
6
3
7
7
6
8
5
4
9
7
7
7
8
4
8
6
6
8
8
8
